12/umbraco-deploy/the-licensing-model.md

```diff
@@ -75,6 +77,30 @@ You can view the status of the Umbraco Deploy On-premise license in the backoffi
 
 ![Licenses screen in Umbraco backoffice](./images/licenses-screen.png)
```

Comment:
seems the image is not working

Reply:
It seems OK locally (i.e. I have the file at `\UmbracoDocs\12\umbraco-deploy\images\licenses-screen.png`).  And I've not actually added this in this PR.